ChromeExtension/js/help.js

```diff
@@ -1,12 +1,19 @@
-document.addEventListener("DOMContentLoaded", function() {
-
-    $('#instructions').on('click', 'a', function() {
-        chrome.tabs.create({
-            url: $(this).attr('href')
-        });
-        return false;
-    });
-
-
-
-});
+document.addEventListener("DOMContentLoaded", function() {
+
+    $('#instructions').on('click', 'a', function() {
+        chrome.tabs.create({
+            url: $(this).attr('href')
+        });
+        return false;
+    });
+
+    if (localStorage["defaultGroup"] != null && localStorage != "") {

```

Comment:
You seem to do this in a few places create a method for it. Replace the uses with the method
localStorage["defaultGroup"] != null && localStorage != ""

Could be something like localStorageHasValue("defaultGroup")